plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcColumnHandle.java

```diff
@@ -108,6 +109,11 @@ public ColumnMetadata getColumnMetadata()
                 .build();
     }
 
+    public ColumnBasicMetadata getColumnBasicMetadata()
+    {
+        return new ColumnBasicMetadata(columnName, columnType, nullable, false);
```

Comment:
why not `getColumnMetadata(). toBasic()`?

Reply:
I wanted to avoid intermediate object creation. Do you think, it is worth to create?